tomster-player/app/router.js

```diff
@@ -8,6 +8,6 @@ export default class Router extends EmberRouter {
 
 Router.map(function() {
   this.route('albums', { path: 'library' }, function() {
-    this.route('album', { path: ':album_id'})
+    this.route('album', { path: ':album_id'});
```

Comment:
Not really related to this PR but we should move this change into the routing commit to avoid confusion.

Reply:
Moved the changes to the routing commit